recipes/pytspack/meta.yaml

```
@@ -0,0 +1,40 @@
+{% set name = "pytspack" %}
+{% set version = "0.1" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/noaa-oar-arl/pytspack/archive/v{{ version }}.tar.gz
+  sha256: 3e35e2c56ace4452dc3aadf030f081581da7257044cd66f24c3343f396cdca1e
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+
+requirements:
+  build:
+    - {{ compiler('fortran') }}
+  host:
+    - python
+    - pip
+    - numpy
+  run:
+    - python
+    - numpy
```

are you compiling agains ``numpy``? if you are, it should be
```suggestion
    - {{ pin_compatible('numpy') }}
```

---

This requires numpy to be installed as it uses f2py to compile the fortran code using numpy.distutil.  Is there a better way to do this?

---

fixed